core/src/edu/asu/c3simulator/screens/BusinessDirectonScreen.java

```diff
@@ -0,0 +1,159 @@
+package edu.asu.c3simulator.screens;
+
+import java.util.LinkedList;
+import java.util.List;
+
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.scenes.scene2d.Actor;
+import com.badlogic.gdx.scenes.scene2d.Stage;
+import com.badlogic.gdx.scenes.scene2d.ui.ButtonGroup;
+import com.badlogic.gdx.scenes.scene2d.ui.CheckBox;
+import com.badlogic.gdx.scenes.scene2d.ui.Label;
+import com.badlogic.gdx.scenes.scene2d.ui.Skin;
+import com.badlogic.gdx.scenes.scene2d.ui.Table;
+import com.badlogic.gdx.scenes.scene2d.ui.VerticalGroup;
+import com.badlogic.gdx.scenes.scene2d.utils.Align;
+
+import edu.asu.c3simulator.widgets.Location;
+
+/**
+ * Displays two options for general company goals and highlights selected option.
+ * 
+ * @author Alyahya, Mohammed some of Zack's code was reused here
+ */
+public class BusinessDirectonScreen extends BusinessCreationGuideScreens
+{
+	private Skin skin;
+	private ButtonGroup optionCheckBoxesGroup;
+	private CheckBox longTermOption, shortTermOption;
+	
+	public BusinessDirectonScreen(Stage stage)
+	{
+		this.skin = new Skin(Gdx.files.internal("skins/default/uiskin.json"));
+		Table directions = new Table();
+		
+		Actor longTermDirection = createLongTermDirection();
+		Actor shortTermDirection = createShortTermDirection();
+		
+		longTermOption = new CheckBox(" Long Term", skin);
+		shortTermOption = new CheckBox(" Short Term", skin);
+		
+		Label longTermLabel = new Label("Long Term", skin);
+		Label shortTermLabel = new Label("Short Term", skin);
+		
+		optionCheckBoxesGroup = new ButtonGroup(longTermOption, shortTermOption);
+		optionCheckBoxesGroup.setMaxCheckCount(1);
+		optionCheckBoxesGroup.setMinCheckCount(0);
+		optionCheckBoxesGroup.setUncheckLast(true);
+		
+		directions.add(longTermLabel).height(100).top();
+		directions.add(shortTermLabel).height(100).top().spaceLeft(75);
+		directions.row();
+		directions.add(longTermDirection).top();
+		directions.add(shortTermDirection).top().spaceLeft(75);
+		directions.row();
+		directions.add(longTermOption).height(100);
+		directions.add(shortTermOption).height(100).spaceLeft(75);
```

Comment:
Is this pattern of 100, 75 coincidental, or systematic?
If the latter, let's make them constants, either local or class scoped.